app/assets/stylesheets/layout_components/card.scss

```diff
@@ -0,0 +1,41 @@
+.card {
+  background: $white;
+  padding: $base-padding 0;
+  text-align: center;
+  margin-bottom: $base-margin;
+
+  &__title {
+    font-size: $base-font-size*1.45;
+    padding: 7px;
+    font-weight: bold;
+    text-align: center;
+
+    @include breakpoint(small only) {
+      font-size: $base-font-size*1.25;
+    }
+  }
+
+  &--blue-bg {
+    @extend .card__title;
+    background: $brand-blue-light;
+    color: $white;
+    position: relative;
+  }
+
+  &__image {
+    margin-bottom: $base-margin;
+  }
+
+  &__content-bg {
+    background: $ash;
+    border-top: 2px solid $stone;
+    border-bottom: 2px solid $stone;
+    padding: $base-padding 0;
+    margin-bottom: $base-margin;
+  }
+
+  &--content {
+    padding: 0 $base-padding;
```

Comment:
can we use columns for the left and right padding here? do we need to zero out the top and bottom?

Reply:
same as what I was doing above, i'll change this.